windows/internal/cuda_install.bat

```diff
@@ -87,6 +88,22 @@ if not exist "%SRC_DIR%\temp_build\cudnn-10.2-windows10-x64-v7.6.5.32.zip" (
     set "CUDNN_SETUP_FILE=%SRC_DIR%\temp_build\cudnn-10.2-windows10-x64-v7.6.5.32.zip"
 )
 
+:cuda110
+
+if not exist "%SRC_DIR%\temp_build\cuda_11.0.1_451.22_win10.exe" (
+    curl -k -L http://developer.download.nvidia.com/compute/cuda/11.0.1/local_installers/cuda_11.0.1_451.22_win10.exe --output "%SRC_DIR%\temp_build\cuda_11.0.1_451.22_win10.exe"
```

Comment:
I can upload it to S3 if you want.

Reply:
Feel free to do so if you prefer S3, and I will update the link here. Also, we just get a link for cuDNN as well, see: https://github.com/pytorch/builder/pull/463

Reply:
http://developer.download.nvidia.com/compute/cuda/11.0.1/local_installers/cuda_11.0.1_451.22_win10.exe
https://developer.download.nvidia.com/compute/redist/cudnn/v8.0.0/cudnn-11.0-windows-x64-v8.0.0.180.zip